layout: page
title: "InTraC knowledge base"
permalink: /intrac

# InTraC knowledge base

Welcome to the InTraC knowledge base. It is the central hub for finding, analysing and disemminating data of the InTraC CRC.
In addition it provides training, service and a community for getting in touch with fellow data analysis experts.
The central components are:

- unified data analysis workflows for genomics data
- training resources to get up to speed with data analysis
- data management in the InTraC knowledge base
- the interactive InTrac data analysis platform
- data analysis services
  - consulting
  - mentoring
  - analysis as a service

## Unified data analysis workflows for genomics data

Bulk sequencing: 
- RNA-Seq ([nf-core](https://nf-co.re/rnaseq/3.22.2/)) & miRNA-Seq
- Assembly and identification or Immunoglobulins from RNA-Seq data (igfinder)
- ChIP-Seq (ChIP-seq, cutandrun, [nf-core[(https://nf-co.re/chipseq/2.1.0/))
- ATAC-Seq ([nf-core](https://nf-co.re/atacseq/2.1.2/))
- WGS variant calling ([Sarek](https://nf-co.re/sarek/3.7.1/), [nf-core](https://nf-co.re/raredisease/2.6.0/))
- Methyl-Seq ([nf-core](https://nf-co.re/methylseq/4.2.0/))
- Crispr-Seq ([nf-core](https://nf-co.re/crisprseq/2.3.0/))

Single cell sequencing:  

- 10x Genomics workflows
  - Single cell variant calling and genomic demultiplexing (vireo, monopogen)
  - Parse Biosciences workflow

Long read sequencing:
- scnanoseq (nf-core)
- Parse Biosciences workflow

## Training resources to get up to speed with data analysis

## Data management in the InTraC knowledge base

## The interactive InTrac data analysis platform

## Data analysis services
